packages/most-gestures/src/drags.js

```diff
@@ -0,0 +1,84 @@
+const { merge } = require('most')
+
+// based on http://jsfiddle.net/mattpodwysocki/pfCqq/
+function mouseDrags (mouseDowns$, mouseUps, mouseMoves, settings) {
+  const {pixelRatio} = settings
+  return mouseDowns$.flatMap(function (md) {
```

Comment:
What's the significance of the "$" at the end of the variable name?

Reply:
Ok. I’ll add some notes about the fixes

Reply:
No idea about the choice of variable names. I suggest we leave all the code as taken, except the changes, and hope there’s a merge to the original repository later.

Reply:
hah, no problem. I thought it might be some convention that I'd never run into before.